src/test/librbd/managed_lock/test_mock_AcquireRequest.cc

```diff
@@ -136,7 +136,7 @@ TEST_F(TestMockManagedLockAcquireRequest, Success) {
   C_SaferCond ctx;
   MockAcquireRequest *req = MockAcquireRequest::create(mock_image_ctx.md_ctx,
      mock_image_ctx.image_watcher, ictx->op_work_queue, mock_image_ctx.header_oid,
-     TEST_COOKIE, &ctx);
+     TEST_COOKIE, true, &ctx);
```

Comment:
Need test case(s) for shared lock to verify correct setting is getting passed to the cls method